File: term2/week-c/exercise3.c
#include <stdio.h>

int main() {
    int start, stop, increment;

    printf("Enter the start number: ");
    scanf("%i", &start);

    printf("Enter the stop number: ");
    scanf("%i", &stop);

    printf("Enter the increment amount: ");
    scanf("%i", &increment);

    for (int i = start; i < stop+1; i = i + increment)
    {
        printf("%i\n", i);
    }
}

File: term2/week1/week1-rs/exercise3.rs
use std::io;

fn main() {

    let mut start = String::new();
    println!("Enter the start number: ");
    io::stdin()
        .read_line(&mut start)
        .expect("Failed to read, get good");
    let start = start.trim().parse::<i32>().unwrap();

    let mut stop = String::new();
    println!("Enter the stop number: ");
    io::stdin()
        .read_line(&mut stop)
        .expect("Failed to read, get good");
    let stop = stop.trim().parse::<i32>().unwrap();

    let mut increment = String::new();
    println!("Enter the increment amount: ");
    io::stdin()
        .read_line(&mut increment)
        .expect("Failed to read, get good");
    let increment = increment.trim().parse::<usize>().unwrap();

    println!("\n");
    
    for i in (start..stop).step_by(increment) {
        println!("{}", i)
    }
    println!("{}", stop);
}